web/src/features/time/TimeController.tsx

```diff
@@ -112,6 +112,7 @@ export default function TimeController({ className }: { className?: string }) {
     },
     [setSelectedDatetime, selectedDatetime.datetime, numberOfEntries, setTimeAverage]
   );
+
```

Comment:
👀

Reply:
Ha, that's intentional. I thought more visual spacing could help differentiate between the logic and JSX of this component